### ISSUE TITLE
Spacing issue not detected in expression-bodied members

### ISSUE THREAD

**Issue by bexxx** [DotNetAnalyzers/StyleCopAnalyzers#1764]
```csharp
namespace NS1
{
    public static class C1
    {
        public static string Title =>Localization.GetLocalizedString("SetupTitle");
    }
}
```

The missing space around `=>` is not identified by the current rules (before and after are not detected).

**Comment by bexxx** [comment]
I'll make a SA1003 out of it + add quick fixes.
